react/modules/react-evolution/README.md

```diff
@@ -236,11 +236,73 @@ approach for testing and getting feedback on new features before stable release.
 
 ### 2024:
 
-Versions: 19 expected
+Versions: 19.0.0 (December 05, 2024)
+(What's new in React 19)[https://react.dev/blog/2024/12/05/react-19#whats-new-in-react-19]
```

Comment:
```
Versions: 19.0.0 (December 05, 2024) 

[What's new in React 19](https://react.dev/blog/2024/12/05/react-19#whats-new-in-react-19)
```

Reply:
What's wrong? :)

Reply:
Currently, it is rendered like this:
![image](https://github.com/user-attachments/assets/c5010c4c-381a-443c-b692-c8d4220c59ad)

And after the suggested changes:
![image](https://github.com/user-attachments/assets/c1b8005d-0c20-4852-b779-09b82305b923)

Reply:
Agree, pushed the changes